Admin/AbstractAdmin.php

```diff
@@ -66,6 +66,7 @@
         )\\\(.*)@x';
 
     const MOSAIC_ICON_CLASS = 'fa fa-th-large fa-fw';
+    const DEFAULT_EXPORT_FORMATS = array('json', 'xml', 'csv', 'xls');
```

Comment:
Three times you mean. On the getter, in the new service, and in the test for the new service